### ISSUE TITLE
Add separate `-q`/`--quiet` flag

### ISSUE THREAD

**Issue by kankaristo** [AUNaseef/protonup#20]
`-y`/`--yes` doesn't usually suppress all output, it just automatically answers "yes" to all prompts.

It would be nice to have a version of `-y`, which shows normal output (to monitor progress, etc.), but still suppress all prompts.

So, maybe a separate "quiet" flag (`-q`/`--quiet`) for suppressing all output (which would also imply `-y`)? This is how e.g. `apt` and `dnf`/`yum` do things.

**Comment by AUNaseef** [comment]
Yeah! I can see this being useful, it would be more intuitive too.
